src/test/kotlin/no/nav/familie/tilbake/behandling/BehandlingServiceTest.kt

```diff
@@ -1402,6 +1406,23 @@ internal class BehandlingServiceTest : OppslagSpringRunnerTest() {
         exception.message shouldBe "Behandling med id=${behandling.id} er allerede ferdig behandlet."
     }
 
+    /* Ny test */
```

Comment:
Hva menes med ny test her? Har vi en 'gammel' variant av denne testen? 
Hvis så, kan den kommenteres ut eventuelt deprecates?

Reply:
Tipper det bare var en midlertidig "tag" for å enkelt kunne finne tilbake til via søk, eller noe sånt :)